api/v1beta2/cryostat_types.go

```diff
@@ -487,12 +488,38 @@ type TemplateConfigMap struct {
 
 // Authorization options provide additional configurations for the auth proxy.
 type AuthorizationOptions struct {
-	// Reference to a secret and file name containing the Basic authentication htpasswd file
+	// Configuration for OpenShift RBAC to define which OpenShift user accounts may access the Cryostat application.
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	OpenShiftSsoRoles *OpenShiftSsoRoles `json:"openShiftSsoRoles,omitempty"`
```

Comment:
This should be `OpenShiftSSORoles`: https://go.dev/wiki/CodeReviewComments#initialisms. We'll also need a displayName here for "OpenShift SSO Roles"